docs/SOPs/contributor_communication_SOP.md

```diff
@@ -263,3 +255,27 @@ Note that this is a constantly changing space, this email was last updated 2020-
 > 
 > On behalf of the HCA DCP Data Wrangling Team
 
+
+### Contributor credentials and upload area
+
+> Dear [name of contributor assigned in the ticket],
+>
+> Attached at the end of the email you will find a zip file with the credentials you need to upload your data.
+> 
+> Your upload area ID is: [Generated upload area UUID]
```

Comment:
I see that not all upload areas are named as UUID. 
We could use a more informative naming pattern like: 

- `contributor-id`-`short-name`  
- `contributor-id`-`short-name`-`uuid`
- `contributor-id`-`uuid`

Reply:
@amnonkhen they are not named as UUID but they all can be accessed using the UUID with the command `hca-util select <uuid>`